### PR TITLE
Update Marathon RAML

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Update Marathon RAML dependencies and align health checks with latest spec. The latest Marathon release includes the following fixes: 

* https://github.com/mesosphere/marathon/issues/4766
* https://github.com/mesosphere/marathon/issues/4767

### CODE REVIEW

**Comment by wavesoft** [comment]
LGTM! ✨

File: plugins/services/src/js/reducers/serviceForm/HealthChecks.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ function mapHealthChecks(item) {
     newItem.protocol = item.protocol;
     if (item.protocol.toUpperCase() === 'COMMAND' && item.command != null) {
       newItem.command = {
-        command: item.command
```
Comment (nLight) [inline plugins/services/src/js/reducers/serviceForm/HealthChecks.js]: we gotta update this in the form too, so the HealthCheck partial looks for errors in `command.command`

Comment (Poltergeist) [inline plugins/services/src/js/reducers/serviceForm/HealthChecks.js]: https://github.com/dcos/dcos-ui/blob/2ffbd89cdab8b53c677650da87e66cd3538b7acf/plugins/services/src/js/components/forms/HealthChecksFormSection.js#L102

Comment (orlandohohmeier) [inline plugins/services/src/js/reducers/serviceForm/HealthChecks.js]: Good catch!

Comment (orlandohohmeier) [inline plugins/services/src/js/reducers/serviceForm/HealthChecks.js]: ✅

**Comment by orlandohohmeier** [comment]
@nLight @Poltergeist I've addressed all issues

**Pull request merged**